bagOfWords.py

```diff
@@ -0,0 +1,61 @@
+import re
+import csv
+from bs4 import BeautifulSoup
+from nltk.tokenize import word_tokenize
+from nltk.corpus import stopwords
+from sklearn.feature_extraction.text import CountVectorizer
+
+def removeNonLetter(sentence):
+	filteredSentence = []
+	sentence = word_tokenize(sentence)
+	for word in sentence:
+		if word.isalpha():
+			filteredSentence.append(word)
+	filteredSentence = " ".join(filteredSentence)
+	return filteredSentence
+
+def preProcessing(sentence):
+	sentence = removeNonLetter(sentence)
+	return sentence
+
+#main program
+fp = open('reviews.csv','r')
+line = fp.readline()
+dataset = []
+for i in range(0,1600):
+	line = fp.readline()
+	dataset.append(preProcessing(line))
+fp.close()
```

Comment:
```suggestion
dataset = []
with open('reviews.csv', 'r') as fp:
    for i in range(0,1600):
        line = fp.readline()
        dataset.append(preProcessing(line))
```
Here I would recommend you to use the `with` context manager to handle opening and closing files. I'm not sure if you have a particular reason to use `range(0, 1600)` to read lines, but unless there is some particular meaning to that number, I would read line by line like this:

```
for line in fp:
    dataset.append(preProcessing(line))
```

https://docs.python.org/3/tutorial/inputoutput.html#methods-of-file-objects

Reply:
Here, the "reviews. csv" was a collection of hotel reviews with each reviews in a line and there were 1600 rows in the file. So I was pulling out the reviews and making another file and that is the code which was serving that purpose.